gitmanager.py

```diff
@@ -88,6 +89,8 @@ def add_to_blacklist(cls, **kwargs):
             # Set up parameters for watch vs blacklist
             if blacklist_file_name in ['watched_keywords.txt']:
                 op = 'watch'
+                now = datetime.now().strftime('%s')
+                item_to_blacklist = "\t".join([now, username, item_to_blacklist])
                 item = item_to_blacklist.split('\t', 2)[2]
```

Comment:
It would be more elegant to save the original `item` from `item_to_blacklist` so you don't have to `split` what you just `join`ed.  But we can fix that up later.